### PR TITLE
deposit: helptexts for fields

### PR DESCRIPTION
**Pull request by mb-wali**
closes inveniosoftware/invenio-app-rdm#453

*screenshots*
publication-date

version

relatedWorks

### CODE REVIEW

File: src/lib/components/RelatedWorksField.js [hunk 1]
```diff
@@ -26,6 +27,10 @@ export class RelatedWorksField extends Component {
 
     return (
       <>
+        <label className="helptext">
+          Specify identifiers of related works. Supported identifiers include DOI, Handle, ARK, \
+          PURL, ISSN, ISBN, PubMed ID, PubMed Central ID, ADS Bibliographic Code, arXiv, Life Science Identifiers (LSID), EAN-13, ISTC, URNs, and URLs.'
+        </label>
```
Comment (fenekku) [inline src/lib/components/RelatedWorksField.js:33]: Looking at the screenshot it seems like some bottom padding on this one would help separate it from the ArrayField, no?

**Review by fenekku — COMMENTED** [review]
The Dates field had a helptext before but it's spacing is not quite the same as those ones. We should align all helptext spacing to be consistent. My preference is to reduce the space between an input and the helptext below it (and increase the space when the help text is not just for the field, but for the section).

**Comment by mb-wali** [comment]
> The Dates field had a helptext before but it's spacing is not quite the same as those ones. We should align all helptext spacing to be consistent. My preference is to reduce the space between an input and the helptext below it (and increase the space when the help text is not just for the field, but for the section).

Strange because i did not touch the css of that see [here](https://github.com/inveniosoftware/invenio-app-rdm/pull/502) - only the target.  i will double check.

**Comment by zzacharo** [comment]
@mb-wali my only concern is regarding the "Related works" helptext...I would put some bottom space between the helptext and the field below...

**Comment by mb-wali** [comment]
> @mb-wali my only concern is regarding the "Related works" helptext...I would put some bottom space between the helptext and the field below...

see here - https://github.com/inveniosoftware/invenio-app-rdm/pull/545#issue-568159871 , more than this?

**Comment by mb-wali** [comment]
> @mb-wali my only concern is regarding the "Related works" helptext...I would put some bottom space between the helptext and the field below...

ah i see it, it did not work for related work, i will try to add inline css in react.

**Comment by mb-wali** [comment]
better? 
related-spacing

**Pull request merged**